storage/2017-07-29/table/entities/query.go

```diff
@@ -19,6 +19,9 @@ type QueryEntitiesInput struct {
 	// An optional comma-separated
 	PropertyNamesToSelect *[]string
 
+	// An optional OData top
+	Limit *int
```

Comment:
does this want to be called Top rather than Limit?

Reply:
Hi @tombuildsstuff 
OK, I rename it.